docs/cookbook/using-S3-storage.md

```diff
@@ -0,0 +1,112 @@
+# Using S3 services with Apostrophe
+
+Deploying Apostrophe to production requires that you specify hosting for the code base, the MongoDB instance, and storage for any uploaded content. Many hosting services can provide all three, but it is also possible and sometimes desirable to split these between different services. This tutorial looks at storage solutions using the S3 API.
```

Comment:
This is a 💯 discussion, but maybe "file storage" rather than just "storage" in the last sentence, for clarity about which of the three problems is being solved.

Reply:
Changed.